crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -286,6 +290,13 @@ where
         }
     }
 
+    pub(crate) fn is_link_node(&self) -> bool {
```

Comment:
I'm pretty sure this could be just a `matches!(self.kind, ContainerNodeKind::Link(_))` or something similar.